NetKAN/World-Airforce-Insignia.netkan

```diff
@@ -0,0 +1,11 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "World-Airforce-Insignia",
+    "$kref"        : "#/ckan/github/chaseAEd/World-Airforce-Insignia",
+     "name"         : "World Airforce Insignia",
+    "abstract"     : "A simple mod that adds a range of current, or historic airforce roundels to add to your planes.",
+    "license"      : "MIT",
+    "ksp_version"  : "1.2.99",
```

Comment:
#4942 https://github.com/chaseAEd/World-Airforce-Insignia/releases    <-  Does this look ok?